### PR TITLE
COR-538: next new individual

### PR DESCRIPTION
**Pull request by internetti**
add a "save & continue" button, that redirect to a new form after submitting

### CODE REVIEW

**Comment by ludydoo** [comment]
I think that this should be handled by the backend. If we put a checkbox `<input type="checkbox" name="createAnotherIndividual"/>`,
then in the backend, instead of redirecting the user to the new individual page, we put
```
if r.Form.Get("createAnotherIndividual") == "true" {
  http.Redirect(w, r, http.StatusSeeOther, "/countries/.../individuals/new")
  return
}
```

File: web/templates/individual.gohtml [hunk 1]
```diff
@@ -35,7 +44,7 @@
                 </h1>
                 <form id="individualForm"
                       method="post"
-                      action="/countries/{{.Individual.CountryID}}/individuals/{{if .Individual.ID}}{{.Individual.ID}}{{else}}new{{end}}">
+                      action="/countries/{{.RequestContext.SelectedCountryID}}/individuals/{{if .Individual.ID}}{{.Individual.ID}}{{else}}new{{end}}">
```
Comment (ludydoo) [inline web/templates/individual.gohtml:47]: Why this change ?

Comment (internetti) [reply web/templates/individual.gohtml:47]: because the former version did not work. it kept redirecting to the countries page, without submitting. I think Individual.CountryID is not defined before submit?

**Comment by internetti** [comment]
regarding the redirecting, not against the backend approach per se, but we don't currently have validation

File: web/templates/individual.gohtml (hunk 1 above)
Comment (ludydoo) [reply web/templates/individual.gohtml:47]: It should be, it is set for both new individuals and individuals being updated before rendering the page

**Comment by ludydoo** [comment]
> regarding the redirecting, not against the backend approach per se, but we don't currently have validation

yeah we do have validation, the `individual` handler has this:

```
		// Validate the individual
		validationErrors = apivalidation.ValidateIndividual(individual)
		if len(validationErrors) > 0 {
			render()
			return
		}
```

File: web/templates/individual.gohtml (hunk 1 above)
Comment (internetti) [reply web/templates/individual.gohtml:47]: okay, but it still didn't work, does it work for you with the former version?

**Comment by internetti** [comment]
yeah, I didn't mean that validation is not implemented, just not wished for by product at the moment. anyway, will see what I can do about the redirection.

**Comment by internetti** [comment]
I'll put this on hold, I'd say the product tickets have a higher priority, this is more a nice to have
